sovrin_client/test/cli/test_send_node_services_omitted.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from stp_core.loop.eventually import eventually
+
+from plenum.common.constants import ALIAS, CLIENT_STACK_SUFFIX, SERVICES
+from plenum.common.util import getMaxFailures
+from plenum.test import waits
+
+
+@pytest.fixture(scope='module')
+def tconf(tconf, request):
+    oldVal = tconf.UpdateGenesisPoolTxnFile
+    tconf.UpdateGenesisPoolTxnFile = True
+
+    def reset():
+        tconf.UpdateGenesisPoolTxnFile = oldVal
+
+    request.addfinalizer(reset)
+    return tconf
+
+
+def testNewNodeAddedWhenServicesParamOmitted(be, do, poolNodesStarted,
```

Comment:
The test `testSuspendNodeWhichWasNeverActive` already checks this, why a new test?